src/hash.ts

```diff
@@ -0,0 +1,8 @@
+import bcrypt = require('bcrypt');
```

Comment:
```suggestion
import * as bcrypt from 'bcrypt';
```

se fizer assim o lint reclama?

Reply:
o lint não reclama, deu certo!
valeu, estava mesmo querendo um jeito melhor de importar, tinha tentado de outras formas e ele estava reclamando